plugin/index.js

```diff
@@ -21,21 +21,26 @@ export default function({ types: t }) {
                   reference = BabelInlineImportHelper.transformRelativeToRootPath(reference);
                 }
 
-                const id = path.node.specifiers[0].local.name;
-                const content = BabelInlineImportHelper.getContents(givenPath, reference);
-                const variable = t.variableDeclarator(t.identifier(id), t.stringLiteral(content));
-
-                path.replaceWith({
-                  type: 'VariableDeclaration',
-                  kind: 'const',
-                  declarations: [variable],
-                  leadingComments: [
-                    {
-                      type: 'CommentBlock',
-                      value: ` babel-plugin-inline-import '${givenPath}' `
-                    }
-                  ]
-                });
+                var content = BabelInlineImportHelper.getContents(givenPath, reference);
```

Comment:
is there any reason to go for `var` instead of `const` or `let`?

Reply:
Great point! I have updated to use const for the variables.